Makefile

```diff
@@ -45,7 +47,7 @@ fakepackage: go-algorand
 	misc/release.py --host-only --outdir $(PKG_DIR) --fake-release
 
 test: idb/mocks/IndexerDb.go cmd/algorand-indexer/algorand-indexer
-	go test ./... -coverprofile=coverage.txt -covermode=atomic
+	go test -coverpkg=$(COVERPKG) ./... -coverprofile=coverage.txt -covermode=atomic
```

Comment:
It doesn't look like this relates to this PR, but I can't find a good reason not to have it.

Reply:
I'm not sure where my comment went, but I asked this question as well. It relates to reporting code coverage for new tests, where the test was written in another package.

Reply:
I was trying to ensure I had good coverage of my changes to the postgres.go query builder exercised by handlers_e2e_test.go in a different package and was frustrated they weren't showing up, so I made that change. With a small repo like indexer it didn't cost anything to add the additional coverpkg arguments.